packages/shared/src/hooks/post/useSmartTitle.ts

```diff
@@ -26,8 +26,15 @@ export const useSmartTitle = (post: Post): UseSmartTitle => {
   const { isPlus, showPlusSubscription } = usePlusSubscription();
   const { completeAction } = useActions();
 
-  const key = useMemo(() => [...getPostByIdKey(post?.id), 'title'], [post?.id]);
-  const trialKey = generateQueryKey(
+  const key = useMemo(
+    () => [
+      ...getPostByIdKey(post?.id),
+      'title',
+      showPlusSubscription.toString(),
+    ],
+    [post?.id, showPlusSubscription],
```

Comment:
We can maybe do ` {key: 'title', showPlusSubscription: true},` just so query key does not grow indefinitely

Reply:
Not sure what you mean?